tests/php/test-class-amp-theme-support.php

```diff
@@ -1844,7 +1844,7 @@ public function test_dequeue_customize_preview_scripts() {
 	 */
 	public function test_start_output_buffering() {
 		wp();
-		if ( ! function_exists( 'newrelic_disable_autorum ' ) ) {
+		if ( ! function_exists( 'newrelic_disable_autorum' ) ) {
```

Comment:
The extra space in the `function_exists` check kept causing fatal errors when running phpunit locally, so I fixed it.

Reply:
Good catch.